code/modules/mob/living/living.dm

```diff
@@ -630,7 +630,7 @@ default behaviour is:
 		var/speed = movement_delay()
 		resting = TRUE
 		var/dir = C.true_dir
-		if(ishuman(src) && (dir))// If true_dir = 0(src isn't moving), doesn't proc
+		if(ishuman(src) && !src.weakened && (dir))// If true_dir = 0(src isn't moving), doesn't proc
```

Comment:
Needless `src.`, same in the code above with `src.client` and `src.stats`.
```suggestion
		if(ishuman(src) && weakened && (_dir))// If true_dir = 0(src isn't moving), doesn't proc
```

Reply:
If you're in a situation where you actually need to use src to differentiate vars you're slready writing bad code. This just improves readability and has no effect on speed at compile time.